lms/djangoapps/learner_recommendations/utils.py

```diff
@@ -84,29 +84,17 @@ def _has_country_restrictions(product, user_country):
     return user_country in block_list or (allow_list and user_country not in allow_list)
 
 
-def _parse_course_owner_data(owner):
-    """
-    Helper to parse course owner data.
-    """
-    return {
-        "key": owner.get("key"),
-        "name": owner.get("name"),
-        "logo_image_url": owner.get("logo_image_url")
-    }
-
-
 def course_data_for_discovery_card(course_data):
     """Helper method to prepare data for prospectus course card"""
     recommended_course_data = {}
     active_course_run = [course_run for course_run in course_data.get("course_runs", [])
                          if course_run.get("availability") == "Current"][0]
     if active_course_run:
-        owners = map(_parse_course_owner_data, course_data.get("owners"))
         recommended_course_data.update({
             "uuid": course_data.get("uuid"),
             "title": course_data.get("title"),
             "image": course_data.get("image"),
-            "owners": owners,
+            "owners": course_data.get("owners"),
```

Comment:
Does frontend need the complete owners data? If yes, then your serializer also needs to be updated.

Reply:
Only `key` `name` and `logo_image_url ` is required in owners key